UnofficialCrusaderPatch/Localization/English.txt

```diff
@@ -2,6 +2,31 @@
 //         DEV
 // ======================
 
+ai_resources_rebuy
+{
+"Improve AI restocking of needed goods"
+}
+
+ai_resources_rebuy_descr
+{
+"The AI often doesn't restock goods even if it has the gold for it, leaving its production chains running dry. This option reduces the amount of time the AI waits for its stocks to be empty before restocking the necessary goods."
```

Comment:
The description is really good, but does not show up in the GUI, it is needed for context though.

I rephrased it a tiny bit to this:
The AI often doesn't restock goods even if it has the gold for it, leaving its production chains running dry. This option reduces the amount of time the AI waits for stocks to refill before restocking the necessary goods by buying.